### ISSUE TITLE
Running mvn clean install creates a lot of files in parent folder

### ISSUE THREAD

**Issue by caarlos0** [teamed/qulice#506]
Each time I run `mvn clean install`, a `project:findbug_excludes_SOME_NUMER.xml` file is created at the parent folder.

The contents looks like:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<FindBugsFilter>
   <Match>
      <Or>
         <Class name="/src/it/.*"/>
      </Or>
   </Match>
</FindBugsFilter>
```

Every once in a while I clean those files up, but it's kind of annoying:

```
$ ls project*
project:findbug_excludes_1446682337609964000.xml  project:findbug_excludes_38443475077025.xml
project:findbug_excludes_1447450097609404000.xml  project:findbug_excludes_52548913696264.xml
project:findbug_excludes_236938902230540.xml      project:findbug_excludes_78846565333859.xml

project:
LICENSE.txt  README.md  pom.xml  src/
``` 

I think it should not create those files, at  least, not in the parent folder.

**Comment by krzyk** [comment]
@caarlos0 wasn't it fixed in #441? have you tried `0.13.3` qulice?

**Comment by caarlos0** [comment]
@krzyk I was with an older version. Thanks!
